### ISSUE TITLE
Make Status changes visible from Event messages

### ISSUE THREAD

**Issue by DivergentEuropeans** [DivergentEuropeans/loser-simulator#2]
Status changes from random events should be made visible to the player

Like from below's Gym example, happiness and health decreases but may not be very noticeable by the player. We should add this to our event's **message UX** to make it more clear how the Event changes a player's Statuses

`if (new Random().nextDouble() < 0.03) { 
happinessLevel = happinessLevel + 7; healthLevel = healthLevel + 4; amIDeadYet(getRandomString("Went too hard at the gym.", "Overexertion leading to cardiac arrest.", "Slip and fall accident with gym equipment.", "Severe dehydration during intense workout.", "Fatal accident with heavy weights.", "Electrocution from faulty gym machinery.", "Allergic reaction to the pre-workout.", "Heat stroke during high-intensity training.")); messagePrompt("The endorphins are pumping and you feel great. Today was a " + "very productive workout.", 500); return; 
}`

A simple proposal:

Add changed number + emoji symbol to messages themselves, such as "Your message XXXXXXX +5 😀 +7 ❤ "

Better:

Think of a good, simple UX for it

**Comment by Priyansh61** [comment]
@DivergentEuropeans Just a question here.

instead of adding it directly to the message UX what if we create / leverage simple animations that shows the addition once user clicks outside of the message UX , an increase in health and happiness is displayed to the user. in some sort of increase in both the bars in form of cool animations  Let me get some ideas on this.

But would love to know your opinion regarding the same.

**Comment by DivergentEuropeans** [comment]
@Priyansh61 good idea on the animations but some health/happiness increases or decreases may be difficult to notice. When the user has low movement game activity or their drains aren't very high, their bars won't move as much (for example: if taking cheap actions with limited results). As the bars may move around less radically, there wont be a lot of room to notice an animation. I was thinking of something a little simpler UX. 

I created this.

![image](https://github.com/DivergentEuropeans/loser-simulator/assets/20528015/b4d66508-a7b1-4c1d-9af6-b3af34bbdf33)

here is an alternate

![image](https://github.com/DivergentEuropeans/loser-simulator/assets/20528015/f5979799-8533-4a24-ac29-49a8d620371b)

I think both look good, fine w/ either one. But importantly is a distinction here that events which do not spawn a message do not show, so that leads to a few q's @Priyansh61 @pigust 

a) which mock up is better

b) should we show changes to statuses for events that don't trigger a message? (I don't think so. This can be left intentionally vague and for the player to figure out. Events with messages have variability in status changes, so those should be shown. Events without messages have less variability in status changes, so they can be "felt out" by the player & that is part of the challenge)

**Comment by Priyansh61** [comment]
@DivergentEuropeans After thinking about this for  quite a long time I believe 2nd layout is much better

1. provides more immediate and clear feedback by showing the changes directly on the status bars.
2. might clutter the status bar area if there are too many changes, but it is more concise and visually straightforward.
3. more consistent with the overall user experience.

Wdys lenny should I implemet the 2nd mockup only and raise the PR?
